src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.test.tsx

```diff
@@ -158,9 +185,10 @@ describe('<ProposalDataListItemStructure/> component', () => {
             expect(screen.getByText(testProps.title)).toBeInTheDocument();
             expect(screen.getByText(testProps.summary)).toBeInTheDocument();
             expect(screen.getByText(testProps.status)).toBeInTheDocument();
-            expect(screen.getByText(testProps.date)).toBeInTheDocument();
+            expect(screen.getByText(testProps.date!)).toBeInTheDocument();
+            expect(screen.getByText(testProps.id!)).toBeInTheDocument();
             expect(
-                screen.getByText(addressUtils.truncateAddress(testProps.publisher.address ?? '')),
+                screen.getByText(addressUtils.truncateAddress((testProps.publisher as IPublisher).address)),
```

Comment:
Why is the cast to `IPublisher` needed here?

Reply:
It was needed since the `testProps` were typed, and `publisher` is `IPublisher | IPublisher[]`.

Reply:
I see, makes sense 👍🏻